courses/models.py

```
@@ -0,0 +1,49 @@
+        return self.title
+
+class Session(models.Model):
+    course = models.ForeignKey(Course, on_delete=models.CASCADE, related_name='sessions')
+    title = models.CharField(max_length=200)
+    description = models.TextField()
+    order = models.IntegerField(default=0)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        ordering = ['order']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)